docs/prod-host-setup.md

```diff
@@ -76,9 +80,20 @@ variants of Meltdown can be mitigated by enabling this feature.
 Verification can be done by running:
 
 ```bash
-(grep -q "^Mitigation: PTI$" /sys/devices/system/cpu/vulnerabilities/meltdown && echo "KPTI: SUPPORTED (OK)") || echo "KPTI: NOT SUPPORTED (Recommendation: SUPPORTED)"
+(grep -q "^Mitigation: PTI$" /sys/devices/system/cpu/vulnerabilities/meltdown \
```

Comment:
Thanks for fixing the whitespaces and making the bash scripts look nicer. But can you please put that in a separate commit? This is nice to have, and I won't be blocking the PR on it.